media-libs/mesa/mesa-9999.ebuild

```diff
@@ -81,6 +81,7 @@ REQUIRED_USE="
 		)
 	)
 	llvm? ( ${LLVM_REQUIRED_USE} )
+	video_cards_asahi? ( opencl vulkan )
```

Comment:
vulkan isn't really required to be enabled, is it?
Also opencl, but there's the nuance of needing the same underlying stuff to build that was mentioned on IRC or something?

Reply:
yes, it needs especially
```
-       if use llvm && use opencl; then
+       if use llvm && (use opencl || use video_cards_asahi); then
                PKG_CONFIG_PATH="$(get_llvm_prefix)/$(get_libdir)/pkgconfig"
                # See https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/docs/rusticl.rst
                emesonargs+=(
```
when build without `opencl`